eva/models/storage/batch.py

```diff
@@ -349,6 +349,23 @@ def concat(cls, batch_list: Iterable[Batch], copy=True) -> Batch:
 
         return Batch(frame)
 
+    @classmethod
+    def stack(cls, batch: Batch, copy=True) -> Batch:
```

Comment:
`Batch` already contains `merge_column_wise`, `__add__`, `concat`, `join`, and `combine_batches`. I am not sure if `stack` reproduces the functionality of any of these functions but either way this is a problem. @gaurav274

Reply:
What problem are you referring to?

Reply:
Stack is different from the functions that already exist. We're adding a new dimension to the input arrays, not concatenating batches. A batch of size 8 will become a batch of size 1 with `stack`.